packages/ui/.stylelintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	extends: [ '../../.stylelintrc' ],
+	plugins: [ 'stylelint-plugin-logical-css' ],
+	rules: {
+		'declaration-property-max-values': {
+			// Prevents shorthand left/right values (unclear for RTL)
+			margin: 3,
```

Comment:
Up to three values is safe because the left/right values will be the same.

Reply:
Good call! Added in 5bb72dbefa729f9335ad39d891793c3ab3e002c7.

We can add more as we encounter them.